src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -5381,6 +5381,14 @@ bool CEndlessUsbToolDlg::CanInstallToDrive(const CString & systemDriveLetter, co
 		}
 	}
 
+	// TODO: when this is called from the front page of the application, we do not know whether the boot.zip
+	// will contain eosldr stuff. If it does not, this check is still valid; if it does, we can skip all
+	// this (which is sort of the point). Ideas:
+	// - drop this up-front check, only perform it once the boot.zip is available (ie after a 10GB download)
+	// - perform this check after the to-be-installed image is selected, and hardcode the knowledge that
+	//   (master && date > 2017-02-...) || (>= eos3.2) => eosldr is available
```

Comment:
We'll have to do the dance where we publish the new installer for ISOs but not for website visitors, to avoid a window of time where the new installer version is released but users get 3.1.x images when they run it. But that's fine.